Util/ClassExtractor.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace A5sys\TypeScriptGeneratorBundle\Util;
+
+/**
+ * Copied from by Symfony's ClassMapGenerator which is now deprecated
```

Comment:
does it means this class is deprecated ? does the comment still apply ?

Reply:
Symfony's ClassLoader component (from which the class is copied) is deprecated. Since the class will be deleted in the next version of Symfony I have copy/pasted its code in our bundle to be able to use its functionality even when Symfony 4 will be released.

Reply:
ok !